docs/resources/mws_networks_gcp.md

```diff
@@ -0,0 +1,108 @@
+---
+subcategory: "GCP"
```

Comment:
this creates a strange precedent. do we want it?

Reply:
`databricks_mws_workspaces` and `databricks_mws_networks` would apply to both GCP & AWS, so we will need a new subcategory for both resources?